src/index.ts

```diff
@@ -322,3 +329,226 @@ export function callBrowser(browser: Browser, name: string, args?: any): Promise
     const id = util.uid();
     return _callBrowser(id, browser, name, args, {});
 }
+
+function triggerEvent(name: string, args: any, info: ProcedureListenerInfo): void {
+    if (glob.__eventListeners[name])
+        glob.__eventListeners[name].forEach((cb: any) =>
+            util.promiseResolve(cb(args, info)))
+}
+
+/**
+ * Register an event listener.
+ * @param {string} name - The name of the event.
+ * @param {function} cb - The event's callback.
+ */
+export function on(name: string, cb: EventListener): void {
+    if(arguments.length !== 2) throw 'on expects 2 arguments: "name" and "cb"';
+    if(environment === "cef") glob[IDENTIFIER].then((id: string) => mp.trigger(BROWSER_REGISTER, JSON.stringify([id, name])));
```

Comment:
`on` and `off` don't need this line. This is only to tell the client that this browser has a procedure registered. But with events, the client doesn't care who has a listener.